agent/agent.go

```diff
@@ -156,8 +152,6 @@ func gatherWithTimeout(
 	acc *accumulator,
 	timeout time.Duration,
 ) {
```

Comment:
This function is no longer `gatherWithTimeout` as the `timeout` argument isn't used anywhere.